### ISSUE TITLE
Feature request: support for Impala

### ISSUE THREAD

**Issue by shoyip** [multiprocessio/datastation#260]
I currently need to connect to a Cloudera Impala instance in order to make queries and analyses.
It would be nice to be able to integrate this type of connection in DataStation.
I think it might require some sort of use of the Impala ODBC Connector / Driver.
Thanks!

**Comment by eatonphil** [comment]
Good idea! Looks like we should be able to support this with https://github.com/bippio/go-impala rather than ODBC (which isn't ready for DataStation yet unfortunately (see #246).
